<!DOCTYPE html>
<html lang="en">
<head>
   <meta charset="UTF-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="The University of Washington Nanopore Sequencing Core is run by the Miller Lab, led by Danny E. Miller, MD, PhD, FACMG. Services include DNA/RNA extraction, library preparation, genome sequencing, and data analysis, including methylation calling. Samples are sequenced on an Oxford Nanopore PromethION 24."/>
	<meta name="keywords" content="Danny Miller, Danny E. Miller, long-read sequencing, structural variant, structural variation, unsolved genetic disorder, genetic diagnosis, methylation, Oxford Nanopore Technologies, ONT, Nanopore, LRS, whole-genome sequencing, genetic testing, genomics, genome assembly, diagnostic odyssey, UW-NSC, Nanopore Sequencing Core, Univeristy of Washington"/>
   <title>UW Nanopore Sequencing Core</title>
	<link rel="shortcut icon" type="image/png" href="images/favicon.png">
   <!-- Bootstrap -->
   <link href="css/bootstrap-4.4.1.css" rel="stylesheet">
   <link href="css/styles.css" rel="stylesheet" type="text/css">
</head>

<body>

	<!---Navbar--->

	<nav class="navbar navbar-expand-lg navbar-dark bg-dark sticky-top">
      <a class="navbar-brand" href="index.html" target="_blank" rel="noopener noreferrer"><img src="images/MillerLabLogo-Mark-Reverse.png" alt="Miller Lab" width="60" id="Home"></a>
		<button class="navbar-toggler"
			type="button"
			data-toggle="collapse"
			data-target="#navbarSupportedContent"
			aria-controls="navbarSupportedContent"
			aria-expanded="false"
			aria-label="Toggle navigation">
			<span class="navbar-toggler-icon"></span>
		</button>
      <div class="collapse navbar-collapse" id="navbarSupportedContent" data-toggle="collapse" data-target=".navbar-collapse">
        	<ul class="navbar-nav mr-auto">
          	<li class="nav-item active">
			  		<a class="nav-link text-white" href="#top">Overview
				  	<span class="sr-only">(current)</span></a> </li>
          	<li class="nav-item">
			 		<a class="nav-link text-white" href="#CoreServices">Services</a> </li>
	          <li class="nav-item">
			  		<a class="nav-link text-white" href="#Rates">Rates</a> </li>
   	      <li class="nav-item">
			  		<a class="nav-link text-white" href="#SamplePrep">Sample Preparation</a> </li>
   	      <li class="nav-item">
				  	<a class="nav-link text-white" href="#Shipping">Shipping</a> </li>
        	</ul>
      </div>
   </nav>


	<!---Banner--->

   <header>
		<img class="embed-responsive" src="images/UW-NSC_banner.png" alt="NSC banner" id="top">
   </header>


	<!---Intro--->

	<section>
      <div class="container">
	  	<br><br>
        	<div class="row">
          	<div class="col-lg-8 mb-4 mt-3 offset-xl-1 col-xl-10 text-left lead" style="color:#062E59">
			  		<p>The University of Washington Nanopore Sequencing Core provides DNA and RNA sequencing services on the Oxford Nanopore Technologies platform. For collaborators, we perform library preparation and sequencing from a variety of sample inputs, including blood, saliva, buccal swabs, and cell pellets. We also accept direct DNA or RNA submissions.</p>
					<p>This page provides detailed information on available services and how to prepare, package, and ship your samples. Before submitting samples, we ask that you <a href="mailto:MillerLabSeq@uw.edu?Subject=UW NSC Consultation Request">contact us</a> for a consultation. </p>
					<br>
					<p><strong>Our Sample Submission Process:</strong></p>
					<ul>
						<li>Contact us to discuss your sequencing needs.</li>
						<li>Submit a <a href="https://forms.gle/RQSZ2eH24wfvRUs47" target="_blank" rel="noopener noreferrer">sequencing intake form</a>. Please note that the more completely you fill out the manifest portion of the intake form, the more accurate your estimate will be.</li>
						<li>Look for your cost estimate to be issued via email.</li>
						<li>If you would like to proceed, prepare and ship your samples following the guidelines described below.</li>
						<li>We'll notify you when we have received your samples and provide an estimated timeline for the work.</li>
					</ul>
          	</div>
        	</div>
      </div>
  	</section>


	<!---Services--->


	<section>
		<div class="container" id="CoreServices">
			<hr style="border-color:white">
      </div>
   </section>
	<section>
		<div class="container">
       	<div class="row">
          	<div class="col-lg-8 mb-4 offset-xl-1 col-xl-10">
          		<h3 style="color:#659AD2" class="text-left mt-5">SEQUENCING SERVICES</h3>
					<hr>
			  			<img src="images/NSC_GenomeSequencing.png" class="float-right mb-3 ml-2 shadow" style="width:30em; border-radius:4%" alt="Nanopore read length">
						<p class="text-left">Nanopore sequencing is a third-generation sequencing technology that allows for analysis of DNA and RNA sequences without amplification or modification. As nucleic acids pass through protein nanopores, changes in electric current are detected and subsequently decoded into bases, preserving methylation and modifications to the original sequences. With the PromethION instrument, we generate high-quality datasets with average read lengths of 10–60 kb — suitable for long-read applications such as de novo assembly, structural variant discovery, splice variant discovery, and methylation analysis.</p>

						<h5 style="color:#062E59" class="text-left mt-5"><strong>Cell Culture</strong></h5>
						<p>We offer cell culture services for projects requiring fibroblast or lymphoblastoid cell lines. We can culture cells from patient-derived samples, ensuring high-quality growth and expansion for downstream genetic analysis or sequencing. Our team can provide guidance on cell line creation and maintenance, helping ensure that your project progresses smoothly.</p>

						<h5 style="color:#062E59" class="text-left mt-5"><strong>Nucleotide Extraction and Purification</strong></h5>
						<p>DNA and RNA extraction and purification services are available on material inputs including cell pellets, saliva, buccal swabs, and blood. Please see <a href="#SamplePrep">Sample Preparation</a> below for more detail. Contact us to discuss tissue extractions.</p>

          			<h5 style="color:#062E59" class="text-left mt-5"><strong>Sample QC, Cleanup, Shearing &amp; Size Selection</strong></h5>
			  			<p>For all samples, DNA will be quantified using the Qubit, a NanoDrop will be used to assess possible extraction carryover and RNA content (260:230 and 260:280 ratios, respectively), and the Agilent Femto Pulse system is used to evaluate DNA for quality and fragment length. If necessary based on QC results, sample cleanup, shearing, and/or size selection may be performed at additional cost.</p>

						<h5 style="color:#062E59" class="text-left mt-5"><strong>Library Preparation</strong></h5>
						<p>Library preparation methods depend on the type of samples being sequenced. We typically use a ligation library prep for DNA sequencing and a cDNA prep for RNA sequencing, however we also offer direct RNA and rapid library preps.</p>
						<p>We also accept already-prepared libraries for sequencing, but please be aware that we are unable to perform our typical QC on those and thus cannot guarantee their performance. Results can and do vary based on the quality of the extraction and library preparation methods used. For example, DNA sequencing of samples with protein or RNA carryover may not sequence well.</p>

						<h5 style="color:#062E59" class="text-left mt-5"><strong>Barcoding</strong></h5>
						<p>Using either a rapid or ligation library preparation, we can barcode, pool, and sequence up to 24 samples on a PromethION flow cell. Keep in mind that increasing the number of samples per flow cell will decrease the coverage of each sample. A minimum of 4 samples is required for barcoded runs.</p>

          			<h5 style="color:#062E59" class="text-left mt-5"><strong>Sequencing &amp; Basecalling</strong></h5>
						<p>Sequenced libraries may produce datasets of more than 120 Gb, or ~30–40x coverage of the human genome. The raw signal output for all samples will be basecalled using Dorado. We offer several sequencing strategies on the PromethION platform, including:</p>
				  		<ul class="text-left">
							<li>Whole-genome sequencing</li>
				  			<li>Targeted genome sequencing</li>
				  			<li>cDNA sequencing</li>
				  			<li>Direct RNA sequencing</li>
						</ul>

   	       		<h5 style="color:#062E59" class="text-left mt-5"><strong>Analysis</strong></h5>
				  		<p>One hour of analysis is charged per flow cell. Samples are basecalled and run through our standard in-house analysis pipeline, which calls 5mCG/5hmCG modifications when sequencing native DNA. The calls are provided as tags in the unaligned bam files that are returned when sequencing is complete. Our standard pipeline is optimized for human samples; data for other organisms may be delivered in other formats.</p>
						<p>For an added fee, we also  offer a range of additional analysis services that can be tailored to meet the specific needs of your study. For example, our team can assist with tasks such as calling additional methylation models, filtering and prioritizing structural variants, identifying repeat expansions, analyzing other complex variants, and data interpretation support for researchers looking to analyze their own data.</p>

					<p class="text-left"><strong>We can help you determine which sequencing and analysis services are right for your project. <a href="mailto:MillerLabSeq@uw.edu?Subject=UW NSC Consultation Request">Contact us</a> to set up a consultation and to recieve an estimate. </strong></p>

          	</div>
		  	</div>
      </div>
   </section>


	<!---Rates--->

	<section>
		<div class="container" id="Rates">
		<hr style="border-color:white">
        	</div>
   </section>
	<section>
		<div class="container">
       	<div class="row">
          	<div class="col-lg-8 mb-4 offset-xl-1 col-xl-10">
          		<h3 style="color:#659AD2" class="text-left mt-5">2025–2026 SERVICE RATES</h3>
			  		<p></p>
					<table class="table table-sm">
  						<thead>
    						<tr style="background-color:#062E59" class="text-white">
      						<th class="text-left">Service</th>
    							<th class="text-center">Internal UW*</th>
     							<th class="text-center">External Academic or Nonprofit</th>
     							<th class="text-center">Industry</th>
	    					</tr>
  						</thead>
  						<tbody>
   	 					<tr>
      						<td>Cell Culture</td>
     							<td class="text-center">$116.72</td>
     							<td class="text-center">$134.93</td>
     							<td class="text-center">$148.42</td>
    						</tr>
							<tr>
    						<tr style="background-color:#ECECEC">
								<td>Nucleotide Extraction</td>
     							<td class="text-center">$119.62</td>
     							<td class="text-center">$138.29</td>
     							<td class="text-center">$152.11</td>
    						</tr>
    						<tr>
      						<td>Sample QC</td>
     							<td class="text-center">$55.79</td>
     							<td class="text-center">$64.49</td>
     							<td class="text-center">$70.94</td>
    						</tr>
    						<tr style="background-color:#ECECEC">
      						<td>Sample Cleanup</td>
     							<td class="text-center">$41.99</td>
     							<td class="text-center">$48.54</td>
     							<td class="text-center">$53.40</td>
    						</tr>
    						<tr>
      						<td>Shearing</td>
     							<td class="text-center">$48.85</td>
     							<td class="text-center">$56.47</td>
     							<td class="text-center">$62.11</td>
    						</tr>
    						<tr style="background-color:#ECECEC">
      						<td>Size Selection</td>
     							<td class="text-center">$99.38</td>
     							<td class="text-center">$114.88</td>
     							<td class="text-center">$162.37</td>
    						</tr>
    						<tr>
      						<td>Library Preparation</td>
     							<td class="text-center">$259.72</td>
     							<td class="text-center">$300.24</td>
     							<td class="text-center">$330.27</td>
    						</tr>
							<tr style="background-color:#ECECEC">
      						<td>Flow Cell</td>
     							<td class="text-center">$1,238.95</td>
     							<td class="text-center">$1,432.22</td>
     							<td class="text-center">$1,575.45</td>
    						</tr>
    						<tr>
      						<td>Barcoding – Rapid Prep (per sample)</td>
     							<td class="text-center">$41.77</td>
     							<td class="text-center">$48.29</td>
     							<td class="text-center">$53.12</td>
    						</tr>
    						<tr style="background-color:#ECECEC">
      						<td>Barcoding – Ligation Prep (per sample)</td>
     							<td class="text-center">$84.16</td>
     							<td class="text-center">$97.28</td>
     							<td class="text-center">$107.01</td>
    						</tr>
    						<tr>
      						<td>Barcoding – Pool (4–24 samples per pool)</td>
     							<td class="text-center">$106.24</td>
     							<td class="text-center">$122.81</td>
     							<td class="text-center">$135.09</td>
    						</tr>
    						<tr style="background-color:#ECECEC">
      						<td>Analysis (per hour)</td>
    	 						<td class="text-center">$218.36</td>
    	 						<td class="text-center">$252.42</td>
    	 						<td class="text-center">$277.66</td>
    						</tr>
						</tbody>
					</table>
					<hr>
					<small><p class="text-left">*Staff and faculty at the University of Washington paying with a UW Worktag receive the Internal rate; all others are charged the External or Industry rate. Rates are subject to change without notice.</p></small>
          	</div>
		  	</div>
      </div>
   </section>


	<!---Sample Prep instruction--->

	<section>
		<div class="container" id="SamplePrep">
			<hr style="border-color:white">
      </div>
   </section>
	<section>
		<div class="container">
       	<div class="row">
          	<div class="col-lg-8 mb-4 offset-xl-1 col-xl-10">
          		<h3 style="color:#659AD2" class="text-left mt-5">SAMPLE PREPARATION</h3>
					<hr>
					<p>Genome coverage and read length distributions in your aligned data are highly dependent on the quality of input material. The following guidelines will help ensure you receive the highest quality data possible.</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>Accepted Sample Types</strong></h5>
					<p>We currently accept cell pellets, saliva samples, buccal swabs, and blood sample inputs, as well as DNA and RNA submissions. For other sample types, please inquire directly. Follow the guidelines below for your specific sample type.</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>Cell Pellets</strong></h5>
					<p>Please submit between <strong>1 x 10e5 and 1 x 10e7 cells per pellet</strong>. Cells must be frozen as pellets in conical tubes. Microcentrifuge tubes should be double-packed into larger conical tubes and shipped in a polystyrene box filled at least halfway with dry ice. Send these samples using next-day delivery service, for delivery between 10:00 am – 4:00 pm.*</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>Saliva and Buccal Swabs</strong></h5>
					<p>Saliva and buccal swabs collected using DNA Genotek kits can be shipped at room temperature in padded envelopes. Our current extraction protocol uses <a href="https://www.dnagenotek.com/row/products/reagents-preparation/prepIT/PT-L2P.html" target="_blank" rel="noopener noreferrer">PrepIT.L2P</a>.</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>Blood</strong></h5>
					<p>Our recommendation for stabilizing blood samples is to use a <a href="https://www.qiagen.com/us/products/discovery-and-translational-research/sample-collection-stabilization/dna/paxgene-blood-dna-tubes/" target="_blank" rel="noopener noreferrer">PAXgene blood DNA tube</a>. PAXgene tubes can be shipped to us at room temperature within one week of collection. They can be shipped frozen within 2 months of collection. Frozen PAXgene tubes should be shipped horizontally. </p>
					<p>If PAXgene tubes are not available, blood must be stabilized with EDTA, citrate, or heparin in a vacutainer. If possible, blood should <em>not</em> be frozen. If blood must be frozen and shipped, it should be flash frozen quickly after collection and shipped on dry ice.</p>
					<p>Please inquire directly about pick up or drop off of fresh samples in the Seattle area.</p>
					<p>Our current extraction protocol uses the <a href="https://www.neb.com/products/t3050-monarch-hmw-dna-extraction-kit-for-cells-and-blood#Product%20Information" target="_blank" rel="noopener noreferrer">Monarch&reg; HMW DNA Extraction Kit for Cells &amp; Blood</a> for frozen blood with <strong>500 µL to 2 mL</strong> input or the <a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/genomic-dna/puregene-kits" target="_blank" rel="noopener noreferrer">Puregene Blood Kit</a> for fresh blood with 2+ mL of input.</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>DNA</strong></h5>
					<p>For DNA submission, please run a standard or pulsed-field gel, BioAnalyzer, or TapeStation to visualize DNA quality and length before submitting. If your sample looks degraded, re-extract the sample or plan for shorter read lengths. While ~30–40x coverage of the human genome may be achieved with one PromethION flow cell, poor quality DNA will not yield these results. If submitting old, highly fragmented, or otherwise degraded DNA, you should expect ~10–20x coverage.</p>
					<p>Additional guidelines:</p>
					<ul>
						<li>Total material requirements for whole-genome sequencing are <strong>6 µg</strong>.</li>
 						<li>Minimum volume is <strong>10 µL</strong>.</li>
						<li>Ideal concentration range is 60 ng/µL to 120 ng/µL.</li>
 						<li>For best results, use a high molecular weight DNA extraction protocol.</li>
						<li>Do not use DNA isolation protocols containing glycogen.</li>
						<li>Extraction methods should preserve long fragment lengths and avoid repetitive mechanical pipetting or spin columns.</li>
 						<li>We strongly recommend using an RNase during DNA extraction.</li>
						<li>Ideal submissions are RNA- and protein-free.</li>
 						<li>Ensure that your elution buffer does not contain residual denaturants (salts, phenol) or detergents (Triton, SDS).</li>
 						<li>Avoid freeze&ndash;thaw cycles, and do not expose to high temperatures (&#62;65&deg;C) for more than 1 hour.</li>
 					</ul>
			  		<p>Ship DNA in LoBind microcentrifuge tubes or LoBind PCR plates (Eppendorf). Pad and insulate your samples to minimize shearing during shipping.</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>RNA</strong></h5>
					<p>For direct RNA submission, please <a href="mailto:MillerLabSeq@uw.edu?Subject=UW NSC Consultation Request">contact us</a> to discuss the guidelines.</p>

					<h5 style="color:#062E59" class="text-left mt-5"><strong>Still have questions?</strong></h5>
					<p>We’re happy to help! Just send us an <a href="mailto:MillerLabSeq@uw.edu?Subject=UW NSC Consultation Request">email</a>.</p>
          	</div>
		  	</div>
      </div>
   </section>


	<!---Shipping--->

	<section>
		<div class="container" id="Shipping">
			<hr style="border-color:white">
      </div>
   </section>
	<section>
		<div class="container">
       	<div class="row">
          	<div class="col-lg-8 mb-4 offset-xl-1 col-xl-10">
          		<h3 style="color:#659AD2" class="text-left mt-5">SAMPLE DELIVERY</h3>
					<hr>
					<p>If you would like to drop off your samples in person, we accept delivery 9 am – 11 am on weekdays.</p>
					<p>For shipping, please plan for weekday delivery only, 10:00 am – 4:00 pm.* All samples should be delivered to:</p>
			  		<p class="ml-5">University of Washington Nanopore Sequencing Core
			  			<br>c/o Joy Goffena
			  			<br>1705 NE Pacific St.
			  			<br>HSB RM H-458
			  			<br>Seattle, WA 98195</p>
					<p><strong>For all deliveries</strong>:
						<ul>
							<li>Email us with your expected delivery date, tracking number, and digital version of your <a href="https://docs.google.com/spreadsheets/d/e/2PACX-1vQRdopwvstR9OcZdygVceEtkhlLF_UDuHp_uu5RfJ5Ot9gM1u1GNP4MoTi_auR69A/pub?output=xlsx">sample manifest</a>.</li>
							<li>In your delivery package, include your sample manifest printed at a readable size. Please include your name, institution, and email address on the manifest.</li>
						</ul>
					</p>
			  		<p class="mt-4" style="color:darkred"><em>* Please do not send packages for first morning delivery. FedEx and UPS do not have early morning access to our building, so deliveries arriving prior to 9 am, including FedEx First Overnight deliveries, may be delayed by a day or more.</em></p>
			  		<p style="color:darkred"><em>* Please do not send packages to arrive on weekends or after 4:00 pm on weekdays.</em></p>
			  		<p style="color:darkred"><em>* Please do not ship samples without contacting us first.</em></p>
          	</div>
		  	</div>
      </div>
   </section>



<!---Footer--->

  	<footer class="text-left" style="background-color:#ECECEC">
      <div class="container">
        	<div class="row">
          	<div class="text-center col-sm-12 col-lg-2 mt-5 mb-5">
			 		<img alt="Miller Lab logo" style="height: 140px;" src="images/MillerLabLogo-Mark1.png" data-holder-rendered="true">
				</div>
          	<div class="col-sm-12 col-lg-10 mt-1 mb-5">
			  	<br><br>
              	<h6><strong>MILLER LAB</strong></h6>
					<p style="color:#6F6D6D">Division of Genetic Medicine, Department of Pediatrics
					<br>Department of Laboratory Medicine &amp; Pathology
					<br>University of Washington
			  		</p>
		 			<p><font color="#929697">Copyright © 2025 Miller Laboratory. All rights reserved.</font></p>
          	</div>
        	</div>
      </div>
   </footer>



   <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
   <script src="js/jquery-3.4.1.min.js"></script>
   <!-- Include all compiled plugins (below), or include individual files as needed -->
   <script src="js/popper.min.js"></script>
   <script src="js/bootstrap-4.4.1.js"></script>
</body>
</html>